Makefile

```diff
@@ -36,6 +36,20 @@ ifneq ($(GIT_TAG),)
 	VERSION_METADATA =
 endif
 
+# Setting container tool
+DOCKER_CMD := $(shell command -v docker)
+PODMAN_CMD := $(shell command -v podman)
+
+ifdef DOCKER_CMD
+	CONTAINER_TOOL = 'docker'
+else ifdef PODMAN_CMD
+	CONTAINER_TOOL = 'podman'
+endif
+
+ifndef CONTAINER_TOOL
```

Comment:
Also I have rebased the branch on top of main, we are using rebases instead of merges in this repo's commit history.

Reply:
Done! WDTY?

Reply:
Sorry, there's one more thing we missed. You need the `shell` prefix before `command`

Reply:
That's the reason it was failing earlier.

Reply:
@rmarting Thanks for making all the requested changes.
I am just debugging why `make` is saying it can't find `command`
Seems other people had the same issue https://github.com/cnabio/duffle/issues/502
Will merge after I fix this.

Reply:
hmm https://stackoverflow.com/questions/12989869/calling-command-v-find-from-gnu-makefile/12991757

Reply:
also https://stackoverflow.com/questions/5618615/check-if-a-program-exists-from-a-makefile